LaunchAtLogin/copy-helper.sh

```diff
@@ -1,7 +1,15 @@
 #!/bin/bash
 
-origin_helper_path="$BUILT_PRODUCTS_DIR/$FRAMEWORKS_FOLDER_PATH/LaunchAtLogin.framework/Resources/LaunchAtLoginHelper.app"
-helper_dir="$BUILT_PRODUCTS_DIR/$CONTENTS_FOLDER_PATH/Library/LoginItems"
+if [ "${DEPLOYMENT_LOCATION}" = "YES" ] ; then
+    WHERE="${DSTROOT}"
+    helper_dir="${WHERE}/$CONTENTS_FOLDER_PATH/Library/LoginItems"
+    origin_helper_path="${helper_dir}/LaunchAtLogin.framework/Resources/LaunchAtLoginHelper.app"
+else
+    WHERE="${BUILT_PRODUCTS_DIR}"
+    origin_helper_path="${WHERE}/$FRAMEWORKS_FOLDER_PATH/LaunchAtLogin.framework/Resources/LaunchAtLoginHelper.app"
+    helper_dir="${WHERE}/$CONTENTS_FOLDER_PATH/Library/LoginItems"
```

Comment:
Use tab-indentation.

Reply:
This one is the same for both branches, so it could be outside the if-statement.

Reply:
Unfortunately, it can't be set outside the `if` bc it's used inside the `if` to set `origin_helper_path` (and again outside to set `helper_path`), but it also depends on the value of `WHERE` which is set differently in the `if` and `else` cases.

Perhaps you have a different setup in mind?

Reply:
Unfortunately, it can't be set outside the `if` bc it's used inside the `if` to set `origin_helper_path` (and again outside to set `helper_path`), but it also depends on the value of `WHERE` which is set differently in the `if` and `else` cases.

Perhaps you have a different setup in mind?